feature-parts/src/main/java/com/example/feature_parts/detailed_part/DetailedPartFragment.kt

```diff
@@ -51,16 +53,14 @@ internal class DetailedPartFragment : BottomSheetDialogFragment() {
 
         val adapter = DetailedPartAdapter()
 
+        binding.detailedPartButton.setOnClickListener {
+            openBrowser(viewModel.searchQuery)
```

Comment:
А почему нельзя сразу так?
```suggestion
            openBrowser(viewModel.detailedPartLiveData.value?.searchQuery.orEmpty())
```

Reply:
Если не нравится, что некрасиво, можно это сделать в геттер функции для `viewModel.searchQuery`